qubes/tests/ext.py

```diff
@@ -322,3 +330,38 @@ def test_012_supported_services_remove(self):
         self.assertEqual(self.features, {
             'supported-service.test2': True,
         })
+
+    def test_013_feature_set_dom0(self):
+        self.test_base_dir = '/tmp/qubes-test-dir'
+        self.base_dir_patch = mock.patch.dict(
+            qubes.config.system_path, {'dom0_services_dir': self.test_base_dir})
+        self.base_dir_patch.start()
```

Comment:
You need to stop the patch somewhere. For example add `self.addCleanup(self.base_dir_patch.stop)`